test/test-remote/test_prom_remote_write.ts

```diff
@@ -59,7 +59,7 @@ async function queryCortex(baseUrl: string, queryParams: URLSearchParams) {
     even when it sends a JSON document in the response body. Submit a bug
     report, and at some point test that this is not the case anymore here.
   */
-  const url = `${baseUrl}/api/v1/query_range`;
+  const url = `${baseUrl}/api/v1/${queryType}`;
```

Comment:
maybe rename to `queryUrlSuffix`? :shrug:  :-)

Reply:
Done